Mathlib/RingTheory/MvPowerSeries/Evaluation.lean

```diff
@@ -122,6 +122,11 @@ theorem _root_.MvPolynomial.toMvPowerSeries_denseRange :
     simp [coeff_trunc', hdn]
   exact mem_closure_of_tendsto this <| .of_forall fun _ ↦ mem_range_self _
 
+lemma hasEval_X_comp {τ : Type*} {f : σ → τ} (hf : Tendsto f cofinite cofinite) :
```

Comment:
As we discussed, a class that says `Tendsto _ cofinite cofinite` would be useful.